.github/workflows/concrete_python_tests_linux.yml

```diff
@@ -80,6 +80,9 @@ jobs:
             set -e
             rm -rf /build/*
 
+            export PYTHON=python3.12
```

Comment:
I think we should run tests on the  oldest version (ideally all). Dev using up-to-date python can introduce incompatibilites otherwise. E.g. When I use too "new" feature, unsupported in 3.8 the CI usually saves me by being red.

Reply:
or we should use the most-used Python, and it's not 3.12
worst case: CP are tested with 3.8 to 3.12 in CML weeklies.

Reply:
Per https://w3techs.com/technologies/history_details/pl-python/3, the most popular is 3.6 but we don't support it (it's deprecated, https://devguide.python.org/versions/). Then, it's 3.8. So, I would say: unless we go for testing all the versions (which is good, but some work), let's test with 3.8 and let CML test the full stack with all the versions

Reply:
For me 3.8 is the right version due to the reasons @rudy-6-4 mentioned.